README.md

```diff
@@ -26,40 +26,28 @@ likely need to run it locally for other projects in the future.
 
 #### 1) Install edX
 
-Download the edX Vagrant box according to
-[these instructions provided by edX](https://edx-installing-configuring-and-running.readthedocs.io/en/latest/installation/devstack/install_devstack.html#installing-devstack-with-a-direct-vagrant-box-download)
-
-*NOTE: edX now maintains a [Docker-based solution](https://github.com/edx/devstack) for running Open edX, and they have stopped supporting the Vagrant solution. We have not yet done the work necessary to properly
-integrate MicroMasters with Open edX running on Docker in a development context, so Vagrant is still recommended here*
+Download and install the edX Docker containers according to [these instructions provided by 
+edX](https://edx-installing-configuring-and-running.readthedocs.io/en/latest/installation/install_devstack.html)
 
 #### 2) Connect to the VM and run the LMS server
 
 edX has [instructions for this as well](https://edx-installing-configuring-and-running.readthedocs.io/en/latest/installation/devstack/start_devstack.html#connect-to-devstack-vm).
 More concisely, these are the commands you should run to connect to the VM and run the LMS server:
 
-    # In your local edx_devstack/ directory, start the VM
-    vagrant up
-    # Once that's done, ssh into the running VM
-    vagrant ssh
-    # Switch to the edxapp account within SSH session
-    sudo su edxapp
-    # Run the LMS server
-    paver devstack lms
-    # To run the server without updating requirements and compiling assets, add the '--fast' parameter
-    # eg: paver devstack --fast lms
-
-A few notes:
-
-- Switching to the edxapp account sources the edxapp environment and sets the
- current working directory to the edx-platform repository.
-- "LMS" stands for "Learning Management System". The Open edX platform has
- [several different components](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/architecture.html#overview);
- MicroMaster's only depends on LMS.
+    # In your local edx_devstack/ directory, checkout the master branch of devstack
+    git checkout master
+    # assuming you want to run the master branch of edX, check it out
+    make dev.checkout
+    # update devstack, just in case:
+    make down
+    make pull
+    make dev.up
```

Comment:
We shouldn't be repeating the edX instructions here. We should get rid of this section entirely and rename section 1 to `Install and run edX`